app/models/requests/scan.rb

```diff
@@ -25,6 +25,10 @@ def destination
     'SCAN'
   end
 
+  def submit!
+    send_to_symphony_now!
```

Comment:
Good point - I added a test in spec/models/requests/scan_spec.rb. This is also exercised in spec/controllers/scans_controller_spec.rb.